### DISCUSSION TITLE
howto kafka authentication in the helm chart ?

### DISCUSSION THREAD

**Discussion by korenlev** [provectus/kafka-ui#3082] [Q&A] [answered]
howto kafka authentication in the helm chart ?

**Comment by Haarolean** [comment, marked as answer]
[SASL example with docker envs](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-sasl.yaml)
[Charts docs](https://github.com/provectus/kafka-ui/blob/master/charts/kafka-ui/README.md), supporting external configs as well

**Reply by korenlev** [reply]
Tx much
